lib/solr/params.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Solr
+  class Params
+    attr_reader :parameters
+
+    def initialize
+      @parameters = {
+        q: nil,
+        qt: 'search',
+        fq: [],
+        # rows: URIService::DEFAULT_LIMIT,
+        start: 0
+      }
+    end
+
+    def fq(field, value)
+      @parameters[:fq] << "#{field}:\"#{Solr::Utils.escape(value)}\"" unless value.nil? ## should probably escape
```

Comment:
@cgalarza Is this an old comment that can be removed because you're using `Solr::Utils.escape` already?  Or is is indicating something that I'm missing?

Reply:
It can be deleted, i'll remove it in my next PR